docs/api.md

```diff
@@ -20,28 +20,33 @@ When `requestId` mode is used, only `requestId` field is returned back to the br
 It's a safe way to detect a bot server-side without leaking results to the browser.
 This mode is recommended for production usage.
 
-When `allData` mode is used, all data from the bot detection result is returned back to the browser. 
+When `allData` mode is used, all data from the bot detection result is returned back to the browser.
 This mode is not recommended for production, but can be used during development and testing.
 
 
-`endpoint` You don't need to use it, unless you use a subdomain integration. 
+`endpoint` You don't need to use it, unless you use a subdomain integration.
 
 ## `BotDetector.get`
 
 ```js
-botd.get({ tag?: object }): Promise<object>`
+botd.get({ tag?: object }): Promise<object>
 ```
 
-Performs bot detection. Internally it will make an network request to our bot-detection API,
-analyze all signals and return back the `requestId`, which you can use later to retrieve bot detection results.
+Performs bot detection. Internally it will make a network request to our bot-detection API
+and return back the `requestId`, which you can use later to retrieve bot detection results
+using method `BotDetector.poll`. Also, the `requestId` will be set into cookie, so you don't need to think how
```

Comment:
I'm not sure if cookie info should be in this readme and not in the integrations.

Reply:
We save requestId into cookie, then we don't need the parameter for poll() method. 
I think if it's not mentioned, it will be unclear for users how the method understands which requestId is needed.

Reply:
yeah, it makes sense, I'm just not sure why we need `poll` in the browser in the first place if we can just use `allData`.